Core/Core/Modules/ModuleList/ModuleItemCell.swift

```diff
@@ -86,5 +92,20 @@ class ModuleItemCell: UITableViewCell {
         accessibilityIdentifier = "ModuleList.\(indexPath.section).\(indexPath.row)"
         nameLabel.accessibilityIdentifier = "ModuleList.\(indexPath.section).\(indexPath.row).nameLabel"
         dueLabel.accessibilityIdentifier = "ModuleList.\(indexPath.section).\(indexPath.row).dueLabel"
+
+        let isPublishAvailable = ModulePublishInteractor(app: env.app).isPublishActionAvailable
```

Comment:
What do think about moving this initialization to the view controller and passing this value to the cell? We wouldn't create a new interactor for every cell.

Reply:
I planned to refactor this when implementing the business logic but I've done this now. I ended up passing the interactor to these cells since the publish actions eventually need to end up in the interactor.